dev/com.ibm.ws.security.jwt_fat.builder/.classpath

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" path="src"/>
+	<classpathentry kind="src" path="fat/src"/>
+	<classpathentry kind="src" path="test-applications/jwtbuilderclient/src"/>
+	<classpathentry kind="con" path="aQute.bnd.classpath.container"/>
+	<classpathentry combineaccessrules="false" kind="src" path="/com.ibm.ws.security.fat.common"/>
+	<classpathentry combineaccessrules="false" kind="src" path="/com.ibm.ws.security.fat.common.jwt"/>
```

Comment:
These shouldn't be here and should be covered by the bnd.bnd file.

Reply:
Those were leftover from trying to get a usable eclipse workspace.

Reply:
removed via 10539